latch_cli/main.py

```diff
@@ -203,31 +206,76 @@ def init(
 
 
 @main.command("cp")
-@click.argument("source_file", nargs=1)
-@click.argument("destination_file", nargs=1)
-def cp(source_file: str, destination_file: str):
+@click.argument("src", nargs=1)
+@click.argument("dest", nargs=1)
+@click.option(
+    "--max-concurrent-files",
+    "--mcf",
+    help="Maximum number of files to copy concurrently.",
+    type=int,
+    default=4,
```

Comment:
Can you tune these defaults? 4 seems low, but maybe we are maxing out the disc? We might need separate defaults for up and down.

Reply:
the number of concurrent parts is independent of this parameter - this is mostly to control how many progress bars should be shown/how many different files' parts should be copied at once

Reply:
ill set the processpool worker size to that tho